kitchen-tests/kitchen.travis.yml

```diff
@@ -108,3 +111,9 @@ suites:
   - name: end-to-end
     run_list:
       - recipe[end_to_end::default]
+  - name: rspec
+    run_list:
+      - recipe[rspec]
```

Comment:
With the lifecycle hooks couldn't we just copy this file in w/o ever needing a chef cookbook?